All_home_works/hw12/core/__init__.py

```diff
@@ -0,0 +1,9 @@
+from All_home_works.hw12.core.db import URL, Base, Session, engine, session_scope
+
+__all__ = [
```

Comment:
This is required for asterisks import and asterisks import is not recommended. I prefer not to declare this variable at all.